src/handlers/contacts_handler.py

```diff
@@ -170,11 +170,11 @@ def show_all(_: list, book: ContactBook) -> str:
 @capitalize_name
 def add_birthday(args: list, book) -> str:
     name, date = args
-    record = book.find(name)
+    contact = book.find(name)
 
-    if record:
+    if contact:
         try:
-            record.birthday = Birthday(date)
+            contact.birthday = Birthday(date)
             book.save()
             return f"Birthday added for {name}."
         except ValueError:
```

Comment:
I am not sure that we need this Try Except construct. Because it will catch exception before our @input_error decorator and we will miss all our colorama editing